### DISCUSSION TITLE
[BUG/New Feature] - Need to add a flag to cli so that by default LMT will NOT run on interfaces that do NOT support independent sampler..

### DISCUSSION THREAD

**Discussion by caggio** [opencomputeproject/ocp-diag-pci_lmt#7] [Ideas]
Without Independent Sampler, LMT adds real simulated noise to the interface and introduces errors that are NOT distinguishable from real errors. This can be especially impactful in a production environment. 

propose to use --enable_disruptive_test as the flag. 

Note that this mode would not make use of the data gathered by LMT for errors as the errors would be reported by the normal error reporting.

**Comment by sksekar** [comment]
This is implemented in https://github.com/opencomputeproject/ocp-diag-pci_lmt/pull/16
